src/Analytics.js

```diff
@@ -156,6 +174,9 @@ class Analytics {
             return;
         }
         this._paq.push(['setCustomUrl', getRedactedUrl()]);
+        if (typeof this.generationTimeMs === 'number') {
+            this._paq.push(['setGenerationTimeMs', this.generationTimeMs]);
```

Comment:
Saving this as a member variable seems like quite an error prone API: is `trackPageChange()` called anywhere else (or will it be in the future?) If so, if the caller doesn't also use the page change timer, it will end up submitting whatever the last page change timer value was.

Reply:
True that we'd leak incorrect generation times when we don't use both in conjunction. 

We could set it to `null` after it's been sent?

Reply:
See above, but nulling it out is probably acceptable.